cylc/flow/flow_mgr.py

```diff
@@ -55,36 +62,65 @@ def add_flow_opts(parser):
     )
 
 
-def stringify_flow_nums(flow_nums: Set[int], full: bool = False) -> str:
-    """Return a string representation of a set of flow numbers
+def get_flow_nums_set(flow: List[str]) -> FlowNums:
```

Comment:
I wonder if there is sufficient complexity to justify making a class `Flows` to gather up some of this functionality?

Reply:
IMO, a set is fine.

We just need two functions to serialise / deserialise the set for DB / presentation purposes.